stories/Alert.stories.svelte

```diff
@@ -11,7 +11,7 @@ const args = {
 }
 </script>
 
-<Meta title="Atoms/Dialog.Alert" component={Dialog.Alert} />
+<Meta title="Atoms/Dialog.Alert" tags={['autodocs']} component={Dialog.Alert} />
```

Comment:
Is there a way to make this a default so it works on all stories without specifying this tag?

Reply:
Yes there is, thanks for asking.